Tests/Marketplace/search_and_install_packs.py

```diff
@@ -168,8 +168,79 @@ def search_pack(client: demisto_client, prints_manager: ParallelPrintsManager, p
         lock.release()
 
 
+def install_nightly_packs(client, host, prints_manager, thread_index, packs_to_install, request_timeout=999999):
+    """
+    Install content packs on nightly build.
+    Args:
+        client(demisto_client): The configured client to use.
+        host (str): The server URL.
+        prints_manager (ParallelPrintsManager): Print manager object.
+        thread_index (int): the thread index.
+        packs_to_install (list): A list of the packs to install.
+        request_timeout (int): Timeout settings for the installation request.
+
+    Returns:
+
+    """
+    packs_to_install_str = ', '.join([pack['id'] for pack in packs_to_install])
+    message = 'Installing the following packs in server {}:\n{}'.format(host, packs_to_install_str)
+    prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN, include_timestamp=True)
+    prints_manager.execute_thread_prints(thread_index)
+    # make the pack installation request
+    global PACK_INSTALL
```

Comment:
Why is it global if you're using it only in this method?

Reply:
Also, I'd consider renaming it to something more indicative like ALL_PACKS_INSTALLED_SUCCESSFULY or something like that